From 38aec2384de5968b189fe94e838a8d324b0c6cf1 Mon Sep 17 00:00:00 2001
From: auto commit <unknown@unknown>
Date: Fri, 24 Oct 2008 12:22:45 -0700
Subject: [PATCH] cavium: mipsmtregs.h needs hazzards.h

The twisted paths through the incude maze can reveal that you need to
explicitly list/include hazzards.h from within mipsmtregs.h

Signed-off-by: Paul Gortmaker <Paul.Gortmaker@windriver.com>
---
 include/asm-mips/mipsmtregs.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/asm-mips/mipsmtregs.h b/include/asm-mips/mipsmtregs.h
index c9420aa..389712c 100644
--- a/include/asm-mips/mipsmtregs.h
+++ b/include/asm-mips/mipsmtregs.h
@@ -9,6 +9,7 @@
 
 #include <asm/mipsregs.h>
 #include <asm/war.h>
+#include <asm/hazards.h>
 
 #ifndef __ASSEMBLY__
 
-- 
1.5.5.1

